src/bids/datasetParser.js

```diff
@@ -36,8 +36,9 @@ export async function parseBidsSidecar(datasetRoot, relativePath) {
  * @param {string} relativePath The relative path of the file within the dataset.
  * @returns {Promise<[string, {path: string}]>} The file contents and mocked BIDS-type file object.
  */
-async function readBidsFile(datasetRoot, relativePath) {
+export async function readBidsFile(datasetRoot, relativePath) {
```

Comment:
This is only supposed to be used within this file (a `parseBidsTsvFile` function will be implemented above this one, which will also call this one). There's no need for us to parse BIDS files other than those three types.